### PR TITLE
base_splash

### PR DESCRIPTION
**Pull request by DongHien0896**
create base: https://edu-redmine.framgia.vn/issues/1234
create screen splash: https://edu-redmine.framgia.vn/issues/1240

### CODE REVIEW

**Comment by DongHien0896** [comment]
ready

File: app/src/main/java/com/example/dong/moviedb/util/Constant.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.example.dong.moviedb.util;
+
+public class Constant {
```
Comment (tuannguyen38) [inline app/src/main/java/com/example/dong/moviedb/util/Constant.java]: rename to Constants

File: app/src/main/res/drawable-v24/splash_background.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+    <item android:drawable="@color/color_white" />
+
+    <item>
+        <bitmap
+            android:gravity="center"
+            android:src="@drawable/ic_splash" />
+    </item>
+
+</layer-list>
```
Comment (tuannguyen38) [inline app/src/main/res/drawable-v24/splash_background.xml]: need more an endline

**Comment by DongHien0896** [comment]
ready

**Comment by DongHien0896** [comment]
ready

**Comment by DongHien0896** [comment]
ready

**Pull request merged**